### PR TITLE
IDVA6 1180 Add user page

### PR DESCRIPTION
**Pull request by bmchugh-ch**
Link to Jira:
https://companieshouse.atlassian.net/browse/IDVA6-1180

Changes:

Adds the "add user" page including forms, validation and session handling. See IDVA6-1180 for more information.

### CODE REVIEW

File: src/lib/validation/add.user.validation.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { validateEmailString } from "../../lib/validation/email.validation";
+import { validateRole } from "./user.role.validation";
+import * as constants from "../constants";
+
+const FormInputNames = {
+    EMAIL: "email",
+    USER_ROLE: "userRole"
+} as const;
+
+export const addErrorToViewData = (
```
Comment (mmodzelewski-ch) [inline src/lib/validation/add.user.validation.ts:11]: This function could be moved to `src/lib/utils/viewUtils.ts`

File: src/locales/en/translation/add-user.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (mmodzelewski-ch) [inline src/locales/en/translation/add-user.json:1]: There is no Welsh version json created

File: src/locales/en/translation/common.json [hunk 3]
```diff
@@ -3,5 +3,8 @@
     "skip_to_main_content": "Skip to main content",
     "there_is_a_problem": "There is a problem",
     "customer_insights": "Customer insights",
-    "back_link_to_the_previous_page": "Back link to the previous page"
+    "back_link_to_the_previous_page": "Back link to the previous page",
```
Comment (mmodzelewski-ch) [inline src/locales/en/translation/common.json:6]: The Welsh version json is not updated

**Review by mmodzelewski-ch — CHANGES_REQUESTED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mmodzelewski-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**